discord/guild.py

```diff
@@ -1565,15 +1565,36 @@ async def delete(self) -> None:
         guild.
 
         Raises
-        --------
+        -------
         HTTPException
             Deleting the guild failed.
         Forbidden
             You do not have permissions to delete the guild.
         """
-
         await self._state.http.delete_guild(self.id)
 
+    async def set_mfa_required(self, toggle: bool, *, reason: str = None):
```

Comment:
Maybe the `toggle` parameter could be called `required`?

Reply:
I agree

Reply:
```suggestion
    async def set_mfa_required(self, required: bool, *, reason: str = None) -> None:
```